python/azureml/hybrid.py

```diff
@@ -0,0 +1,316 @@
+# ------------------------------------------------------------------------------------------------------
```

Comment:
1. Needs license and copyright header.
2. Rename file to create_hybrid_model_using_azure_ep.py.

Reply:
How about create_hybrid.py?
The sample is only about onnx graph editing, we will have another doc from ort on AzureEP.